src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -20,10 +20,14 @@ package object rlp {
 
   def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
 
+  def encodeToRlp[T](input: T)(implicit enc: RLPEncoder[T]): RLPEncodeable = enc.encode(input)
```

Comment:
@adamsmo you are using to encode `ByteString` only? Was it because you were having trouble?  I think it's using the wrong implicit `ByteArray` -> `Seq` -> `io.iohk.ethereum.rlp.RLPImplicits#toEncodeableList`